ros/src/fastrack/include/fastrack/value/matlab_value_function.h

```diff
@@ -239,7 +225,21 @@ size_t MatlabValueFunction<TS, TC, TD, PS, PC, PD, RS, RD, B>::StateToIndex(
   return idx;
 }
 
-// Accessor for precomputed gradient at the given state.
+// Accessor for precomputed value at the given state's grid cell.
+template <typename TS, typename TC, typename TD, typename PS, typename PC,
+          typename PD, typename RS, typename RD, typename B>
+VectorXd MatlabValueFunction<TS, TC, TD, PS, PC, PD, RS, RD, B>::ValueAccessor(
+    const VectorXd& x) const {
+  // Convert to index and read value one dimension at a time.
+  const size_t idx = StateToIndex(x);
+
+  VectorXd value(x.size());
+  for (size_t ii = 0; ii < value.size(); ii++) value(ii) = data_[ii][idx];
```

Comment:
This won't work. Remember, `data_` is a `std::vector<double>`, i.e. it's flattened. Also `value` output should be a `double`.